src/ucm/bistro/bistro_aarch64.h

```diff
@@ -35,4 +35,17 @@ ucs_status_t ucm_bistro_patch(void *func_ptr, void *hook, const char *symbol,
                               void **orig_func_p,
                               ucm_bistro_restore_point_t **rp);
 
+/* Instruction type */
+typedef uint32_t ucm_bistro_inst_t;
+
+/* Lock implementation */
+typedef struct {
+    ucm_bistro_inst_t b; /* either self branch, or no-op branch */
+} UCS_S_PACKED ucm_bistro_lock_t;
```

Comment:
do we really need a typedef for ucm_bistro_lock_t?

Reply:
I think so because its size depends on platform. (or are you suggesting to replace `ucm_bistro_lock_t` by `struct ucm_bistro_lock`?)

Reply:
ok, I see